File: second.py
print("This program for you are elible or not for vote ")

age = int(input("enter your age  : "))

if age < 18:
    print("you are not eligible ")
elif age > 100:
    print("invailed age ")

elif age > 18:
    print("congratulation you can vote ")

else:
    print("invalid input ")

File: second.java
import java.util.Scanner;

public class second {
    public static void main(String[] args) {

        try (Scanner sc = new Scanner(System.in)) {
            System.out.println("Enter your age : ");
            int age = sc.nextInt();
            if (age < 18) {
                System.out.print("You111 are not eligible to vote");
            } else if (age > 100) {
                System.out.print("Invalid age");
            } else if (age >= 18) {
                System.out.print("Congratulations! You can now vote.");

            } else {
                System.out.print(" invailid input"); // this line will never execute as the previous conditions cover
                                                     // all
                                                     // cases
            }
        }

    }
}
